src/styl/responsive/tablet-portrait.styl

```diff
@@ -47,5 +46,6 @@ $header-height= 50px
 
 #map-holder
   iframe
-    width: 100%
+    position: absolute
```

Comment:
its so it fills the container on safari. ive tested across browsers. not sure why it isnt displaying for you